src/Servers/Kestrel/test/InMemory.FunctionalTests/RequestTests.cs

```diff
@@ -1123,7 +1134,7 @@ public async Task ContentLengthReadAsyncPipeReaderReadsCompletedBody()
         [Fact]
         public async Task ContentLengthReadAsyncSingleBytesAtATime()
         {
-            var testContext = new TestServiceContext(LoggerFactory);
+            var testContext = new TestServiceContext(LoggerFactory, _enableLineFeedTerminator);
```

Comment:
You missed doing this in the above `ContentLengthReadAsyncPipeReaderReadsCompletedBody` test.